src/svd.jl

```diff
@@ -245,6 +245,37 @@ svdvals(A::AbstractVector{T}) where {T} = [convert(eigtype(T), norm(A))]
 svdvals(x::Number) = abs(x)
 svdvals(S::SVD{<:Any,T}) where {T} = (S.S)::Vector{T}
 
+"""
+    rank(S::SVD{<:Any, T}; atol::Real=0, rtol::Real=min(n,m)*ϵ) where {T}
+
+Compute the numerical rank of a `n × m` matrix by counting how many singular values are greater 
+than `max(atol, rtol*σ₁)` where `σ₁` is `A`'s largest calculated singular value.
```

Comment:
The matrix variable `A` should be mentioned earlier. Also, I wonder if there's a way to relate `ε` to the element type of `S`, so that we may avoid mentioning it altogether.

Reply:
In the code itself the `ε` is related to the element type of `S`. I copied the documentation from `generic.jl`

Reply:
The first line should say numerical rank of a matrix A of size `n x m` using its svd `S`